### ISSUE TITLE
I just had an issue where I bought and placed a tent and workbench on the map and as soon as i stepped outside it slingshotted me back to the Sgt. Lake's outpost. I went back to the tent area and it's no longer there.

### ISSUE THREAD

**Issue by Chey1303** [Uuvana-Studios/longvinter-windows-client#566]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Game Version [e.g. 1.0]
 - Steam Version [e.g. 1.0]

**Additional context**

**Comment by Chey1303** [comment]
Update: Tried again to see if it was a one time thing, it was indeed not, happened exactly the same way except this time I could see the workbench and storage container on the other side of the wall in the blank space. It let me click on them and it turned into the bags but wouldn't let me pick them up because that's when it slingshotted me.

**Comment by sirNugg3ts** [comment]
Does this still happen after the update?

**Comment by Chey1303** [comment]
> Does this still happen after the update?
I realized it happens when I place a tent right above the lootable area closest to the Sgt. Lake Outpost. I tested it again to see what would happen on another server and it still happened even after the update.

**Comment by sirNugg3ts** [comment]
can you please give us a recording of this?

**Comment by Chey1303** [comment]
Of course! I'll go see if it's still doing it today and get a recording of it! :)

**Comment by Chey1303** [comment]
Going to try on a few official and non official servers and conclude if it is just a problem on the non official servers.

**Comment by faketucker** [comment]
Tested. This is the area. The tent has disappeared, but everything else you place in the tent remains in the void.

![image](https://user-images.githubusercontent.com/28383557/160506446-2d2a69fe-921f-4d37-87a6-edcac01ef793.png)

**Comment by Mikooboy** [comment]
Moved a bird spawn that would cause weird overlaps here which would delete the house. Should be fixed in the next update
